### PR TITLE
v1.2.1 standard release of fields plugin

### PR DESCRIPTION
**Pull request by rewanth1997**
Hi @corneliusweig @ahmetb ,

This PR updates the versioning and sample examples according to the new release(v1.2.1)

Regards,
Rewanth

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **doesn't** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/fields.yaml is not a straightforward version bump: diff line unrecognized for version bumps: [-      https://github.com/rewanth1997/kubectl-fields/tree/v1.2.0-beta]

File: plugins/fields.yaml [hunk 1]
```diff
@@ -34,14 +34,14 @@ spec:
...
-  version: v1.2.0-beta
+  version: v1.2.1
   homepage: https://github.com/rewanth1997/kubectl-fields
   caveats: |
     Usage:
       kubectl fields <RESOURCE> [options] <PATTERN>
 
     Documentation:
-      https://github.com/rewanth1997/kubectl-fields/tree/v1.2.0-beta
+      https://github.com/rewanth1997/kubectl-fields/tree/v1.2.1
```
Comment (ahmetb) [inline plugins/fields.yaml]: If you change this during a version-bump PR, we can't auto-merge it.

**Comment by corneliusweig** [comment]
@rewanth1997 Can you push again? Looks like the CI had a hickup.

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **doesn't** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file plugins/fields.yaml is not a straightforward version bump: diff line unrecognized for version bumps: [-      https://github.com/rewanth1997/kubectl-fields/tree/v1.2.0-beta]

**Comment by rewanth1997** [comment]
@corneliusweig Done

**Pull request merged**